docs/api/index.rst

```diff
@@ -142,7 +156,12 @@ Time-Series Simulations
    nonlinear_process
    ts_sim
 
+Goodness-of-Fit Simulations
+""""""""""""""""""""""""""""
+.. autosummary::
+   :toctree: generated/
 
+   H0Simulator
```

Comment:
Is this the the `hyppo.tools` module? If not add an `automodule` and `currentmodule` line